PWGGA/GammaConv/AliAnalysisTaskMaterialHistos.cxx

```diff
@@ -320,6 +330,7 @@ void AliAnalysisTaskMaterialHistos::UserCreateOutputObjects()
     fDeDxMapList            = new TList*[fnCuts];
   }
   hNEvents                  = new TH1F*[fnCuts];
+  hBCNumber                 = new TH1F*[fnCuts];
```

Comment:
i recommend to put the if(fDoSelectBCNumber) condition on the creation of this histogram.

Reply:
thanks you for the advice !